pkg/docker/images/drive-harddisk-symbolic.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I don't think it looks super-awesome, but could work. I would allow to clean out one more image, so that's a plus for using the font.